cplusplus.json

```diff
@@ -255,4 +255,39 @@
   "imports": ["ups/upscaledb.h"], "examples": ["https://github.com/cruppstahl/upscaledb/tree/master/samples"]},
+
+  {"id": "cpp.brotli", "name": "Brotli", "tech": ["efficient-data"], "tags": ["compression"], "repo": "google/brotli",
+  "imports": ["brotli/decode.h", "brotli/encode.h"], "examples": ["https://github.com/google/brotli/blob/master/python/_brotli.cc"]},
+  {"id": "cpp.bzip2", "name": "bzip2", "tech": ["efficient-data"], "tags": ["compression"], "repo": "http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz",
+  "imports": ["bzlib.h"], "examples": ["http://www.bzip.org/1.0.5/bzip2-manual-1.0.5.html"]},
+  {"id": "cpp.finite-state-entropy", "name": "FiniteStateEntropy", "tech": ["efficient-data"], "tags": ["compression"], "repo": "Cyan4973/FiniteStateEntropy",
+  "imports": ["FiniteStateEntropy/lib/"], "examples": ["https://github.com/Cyan4973/FiniteStateEntropy/tree/dev/programs"]},
+  {"id": "cpp.physics-fs", "name": "PhysicsFS", "tech": ["efficient-data"], "tags": ["compression"], "repo": "https://hg.icculus.org/icculus/physfs/file",
+  "imports": ["physfs.h"], "examples": ["http://icculus.org/physfs/physfstut.txt"]},
+  {"id": "cpp.karchive", "name": "kArchive", "tech": ["efficient-data"], "tags": ["compression"], "repo": "KDE/karchive",
+  "imports": ["karchive/k"], "examples": ["https://github.com/KDE/karchive/tree/master/examples"]},
+  {"id": "cpp.lz4", "name": "LZ4", "tech": ["efficient-data"], "tags": ["compression"], "repo": "lz4/lz4",
+  "imports": ["lz4.h"], "examples": ["https://github.com/lz4/lz4/tree/dev/examples"]},
+  {"id": "cpp.lzfse", "name": "LZFSE", "tech": ["efficient-data"], "tags": ["compression"], "repo": "lzfse/lzfse",
+  "imports": ["lzfse.h"], "examples": ["https://developer.apple.com/library/content/samplecode/CompressionSample/Listings/CompressionSample_BlockCompression_c.html"]},
+  {"id": "cpp.lzham", "name": "LZHAM", "tech": ["efficient-data"], "tags": ["compression"], "repo": "https://storage.googleapis.com/google-code-archive-source/v2/code.google.com/lzham/source-archive.zip",
+  "imports": ["lzham_static_lib.h"], "examples": ["https://storage.googleapis.com/google-code-archive-source/v2/code.google.com/lzham/source-archive.zip"]},
+  {"id": "cpp.lzmat", "name": "LZMAT", "tech": ["efficient-data"], "tags": ["compression"], "repo": "http://www.matcode.com/lzmat_lib.zip",
+  "imports": ["lzmat.h"]},
+  {"id": "cpp.miniz", "name": "miniz", "tech": ["efficient-data"], "tags": ["compression"], "repo": "https://storage.googleapis.com/google-code-archive-source/v2/code.google.com/miniz/source-archive.zip",
+  "imports": ["miniz.h"], "examples": ["https://storage.googleapis.com/google-code-archive-source/v2/code.google.com/miniz/source-archive.zip"]},
+  {"id": "cpp.minizip", "name": "Minizip", "tech": ["efficient-data"], "tags": ["compression"], "repo": "nmoinvaz/minizip",
+  "imports": ["mz"], "examples": ["https://github.com/nmoinvaz/minizip/blob/master/test/test.c"]},
```

Comment:
while technically correct, I suggest we change this to "mz.h", "mz_". my worry is mz could generate false positives.